src/js/modules/models/pokemon.js

```diff
@@ -0,0 +1,20 @@
+class Pokemon {
+  constructor(id, name, image, type, likes = 121) {
+    this.id = id;
+    this.name = name;
+    this.image = image;
+    this.type = type;
+    this.likes = likes;
+  }
+
+    getLikeSentence=() => {
+      let res = '';
+      if (this.likes <= 1) {
+        res = '1 like';
+      } else {
+        res = `${this.likes} likes`;
+      }
```

Comment:
[optional] You can use a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) in this part of the code to make it more KISS.
It will look like this:
```javaScript
this.likes <= 1 ? res = '1 like' : res = `${this,likes} likes`;
```

Reply:
Hello @enoqueJonas. I think linter doesn't like ternary operator.